sessionctx/variable/sysvar.go

```diff
@@ -799,7 +802,9 @@ var defaultSysVars = []*SysVar{
 		return strconv.FormatInt(int64(GlobalLogMaxDays.Load()), 10), nil
 	}},
 	{Scope: ScopeSession, Name: TiDBPProfSQLCPU, Value: strconv.Itoa(DefTiDBPProfSQLCPU), Type: TypeInt, skipInit: true, MinValue: 0, MaxValue: 1, SetSession: func(s *SessionVars, val string) error {
-		EnablePProfSQLCPU.Store(uint32(tidbOptPositiveInt32(val, DefTiDBPProfSQLCPU)) > 0)
+		enable := uint32(tidbOptPositiveInt32(val, DefTiDBPProfSQLCPU)) > 0
+		EnablePProfSQLCPU.Store(enable)
+		tracecpu.EnablePProfSQLCPU.Store(enable)
```

Comment:
Why also  enable  this ?

Reply:
I will remove one of them.

Reply:
No, you should only set EnablePProfSQLCPU